simplistic_editor/README.md

```diff
@@ -4,81 +4,85 @@ and contract styled ranges of text. For more information visit https://api.flutt
 
-### TextEditingInlineSpanReplacement
+### `TextEditingInlineSpanReplacement`
 A data structure that represents a replacement, with a range, and a generator that produces
-the desired InlineSpan. The generator should return a TextSpan with the desired styling, and the
-range should be the target range for that styling in the current TextEditingValue. This structure also
+the desired `InlineSpan`. The generator should return a `TextSpan` with the desired styling, and the
+range should be the target range for that styling in the current `TextEditingValue`. This structure also
 contains an expand property which dictates if the replacement should continue to expand from the back edge.
 For example say we have "Hello |world|", where "world" is covered by a replacement that bolds the text. If
 the expand property is true, typing text at the back edge of "world|" will expand the range and make any text
-typed also bold. If it is false then the text typed would not be bold. In addition to this property
-this structure also contains methods to expand and contract the range of the replacement according to the
-type of TextEditingDelta it receives, as well as a method to remove a section of the replacement range.
+typed also bold. If it is false then the text typed would not be bold. Additionally, this structure contains
+methods to update the replacement for each subclass of `TextEditingDelta` and a method to remove a section of
+the replacement range.
 
-### ReplacementTextEditingController
-A TextEditingController that manages a list of TextEditingInlineSpanReplacements, that insert custom InlineSpans
-in place of matched TextRanges. The controller syncs the replacement ranges based on the type of TextEditingDelta
-it receives from the BasicTextInputClient, managing any overlapping ranges accordingly. 
+### `ReplacementTextEditingController`
+A `TextEditingController` that manages a list of `TextEditingInlineSpanReplacement`s, that insert custom `InlineSpan`s
+in place of matched `TextRange`s. The controller syncs the replacement ranges based on the type of `TextEditingDelta`
+it receives from the `BasicTextInputClient`, managing any overlapping ranges accordingly.
 
 This controller also contains convenience methods used by the styling toggle buttons toolbar to un-style
 certain ranges, disable the expand property of a replacement, and get the common replacements at a selection
 to determine the current toggle button state.
 
 ## Text Input Layer
-The layer that defines the appearance of a text input field, handles the text input received from 
-the platform, and mutations done by the framework through gestures and keyboard shortcuts.
+The layer that defines the appearance of a text input field, handles the text input received from
+the platform, and mutations done by the framework through gestures and keyboard shortcuts. These classes begin to
```

Comment:
Added this sentence:

> These classes begin to demonstrate the types of logic developers may need if they wish to interact with `TextEditingDelta`s.

Can you confirm that this is correct, @Renzo-Olivares?

Reply:
This is correct!